frontend/src/components/Tag/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Chip } from '@mui/material';
+
+const Tag = ({ label, color }) => {
+  return <Chip label={label} sx={{ backgroundColor: color }} />;
+};
+
+Tag.propTypes = {
+  label: PropTypes.node.isRequired,
+  color: PropTypes.node.isRequired,
```

Comment:
label i color to bardziej PropTypes.string (node to jakbyśmy przekazywali jakiś inny komponent czy coś)